.eslintrc

```diff
@@ -52,6 +61,7 @@
         "plugin:cypress/recommended"
       ],
       "rules": {
+        "react/prop-types": "off",
```

Comment:
What does this rule enforce?

Reply:
Ah this actually isn’t meant to be part of this PR, it won’t be there in the full implementation.

However, in answer to your question, at the moment we type our React components as follows:

```tsx
type Props = {
  someProp: string;
};

const Component = ({ someProp }: Props): React.Component => {
  return <div>Whatever this component does here {someProp}…</div>;
};
```

This is fine-ish, although technically isn’t 100% accurate. React exposes a type called `React.FC` (FC = Functional Component) which is a generic and, using the same example, can be used as follows:

```tsx
Const Component: React.FC<{
  someProp: string;
}> = {
  return <div>Whatever this component does here {someProp}…</div>;
};
```

These are effectively the same thing, but I prefer the terser `React.FC<>` type. It also automatically types the `children` prop for us, which the first example doesn’t.

This `”react/prop-types”` ESLint rule doesn’t allow us to do the second pattern when it’s enabled, so I turned it off while putting this prototype together, purely for my own convenience!